Section 2/2.46_Mixins_Methods_CRUD_2.rb

```diff
@@ -0,0 +1,39 @@
+# require 'bundler/inline'
+ 
+# gemfile true do
+#  source 'http://rubygems.org'
+#  gem 'bcrypt'
+# end
+ 
+require 'bcrypt'
+ 
+users = [
+  { username: "mashrur", password: "password1" },
+  { username: "jack", password: "password2" },
+  { username: "arya", password: "password3" },
+  { username: "jonshow", password: "password4" },
+  { username: "heisenberg", password: "password5" }
+]
+ 
+def create_hash_digest(password)
+  BCrypt::Password.create(password)
+end
+ 
+def verify_hash_digest(password)
```

Comment:
If I see a function named `verify`, I'd expect it to verify. Not return me the value for me to verify externally.

This means the function should be:
```
def verify_hash_digest(password, hash_digest)
  BCrypt::Password.new(password) == hash_digest
end
```